support/vm/invocation_context.go

```diff
@@ -62,6 +67,8 @@ type topLevelContext struct {
 	gasPrices    Pricelist
 	gasUsed      int64
 	gasAvailable int64
+	// Temporary field to workaround test-vector limitations
+	fakeSyscallsAccessed bool
```

Comment:
Both this and RandString's fragile dependence on outside defined schema are things I expect to cover in an upcoming issue about absorbing and moving forward the test-vector format into the specs-actors project.